srcpkgs/serpl/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'serpl'
+pkgname=serpl
+version=0.1.27
+revision=1
+build_style=cargo
+depends="ripgrep"
+short_desc="Simple terminal UI for search and replace"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+license="MIT"
+homepage="https://github.com/yassinebridi/serpl"
+changelog="https://github.com/yassinebridi/serpl/releases"
```

Comment:
They don't seem to publish any release notes there? The release description is always like `Release version x.y`, but doesn't include any release notes.

Reply:
Hm, for template purposes, I'd say even a list of versions is better than nothing